### DISCUSSION TITLE
Browser maximise issue with AWS

### DISCUSSION THREAD

**Discussion by ArvindDangi007** [serenity-bdd/serenity-core#3267] [Q&A]
Hi All,

I am using Serenity BDD, and we are executing our scripts on AWS. However, we are facing a browser maximize issue on the remote machine. We have tried various approaches such as manage().window().maximize(), JavaScript, dimensions, and toolkit, but unfortunately, none of them seem to be working.

Thank you.

**Comment by ArvindDangi007** [comment]
@wakaleo @jeffjensen Could you please help here

**Comment by wakaleo** [comment]
`manage().window().maximize()` should work (if not, it might be a Selenium driver issue). It may be that the resolution configured on the virtual display is smaller than you are expecting. For project-specific support , you also have the option of getting a commercial support package or a one-off troubleshooting session here: https://www.serenitydojo.academy/serenity-bdd-support-packages
